### ISSUE TITLE
No output XML at all

### ISSUE THREAD

**Issue by BroBryce64** [WyattBlue/auto-editor#371]
### Bug description

On the WINDOWS APP, I imported my MP4, was able to get the settings just right, get a really good edit on the preview.  But when I selected the Premiere XML output, and clicked start, I got no output file at all.  Just to be sure, I searched my entire filesystem for the output file, and got nothing.

The command line version has no such issue.

One nice feature might be for the APP to give us a preview of the command line using the options selected so that if the app fails, the command line would be able to do the job in a pinch.

### What's your platform?

```shell
Python Version: 3.11.5 64-bit
Platform: Windows 10 amd64
FFmpeg Version: N-110322-g0c25da5a28-g1440bf15e2+1
FFmpeg Path: C:\Users\BroBryce\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ae_ffmpeg\Windows\ffmpeg.exe
Auto-Editor Version: 23w32a

It's actually Win11:
Edition	Windows 11 Home
Version	22H2
Installed on	‎11/‎22/‎2022
OS build	22621.2134
Experience	Windows Feature Experience Pack 1000.22659.1000.0
```

### What command did you use

```shell
I used the app.
```

### What properties does your input video have?

.\2023-08-25 17-14-24.mp4:
 - video:
   - track 0:
     - codec: h264
     - fps: 30
     - resolution: 1920x1080
     - aspect ratio: 16:9
     - pixel aspect ratio: 1:1
     - duration: 361.766667
     - pix fmt: yuv420p
     - color range: tv
     - color space: bt709
     - color primaries: bt709
     - color transfer: bt709
     - timebase: 1/15360
     - bitrate: 15114453
     - lang: und
 - audio:
   - track 0:
     - codec: aac
     - samplerate: 48000
     - channels: 2
     - duration: 361.706667
     - bitrate: 128237
     - lang: und
 - container:
   - duration: 361.766667
   - bitrate: 15251618 
   
   Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '.\2023-08-25 17-14-24.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf60.3.100
  Duration: 00:06:01.77, start: 0.000000, bitrate: 15251 kb/s
  Stream #0:0[0x1](und): Video: h264 (High) (avc1 / 0x31637661), yuv420p(tv, bt709, progressive), 1920x1080 [SAR 1:1 DAR 16:9], 15114 kb/s, 30 fps, 30 tbr, 15360 tbn (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
  Stream #0:1[0x2](und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 128 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]
Stream mapping:
  Stream #0:1 -> #0:0 (aac (native) -> mp3 (libmp3lame))

**Comment by BroBryce64** [comment]
Also, when I used it to export as a media file, it did not output any file at all, although it did leave the console window open longer, long enough for me to see it running through the frames with FFMPEG.  But it still left no file in my filesystem.  

I tried moving the app folder elsewhere (shorter pathname) and even tried putting the mp4 files in the auto-edit app folder.  Nothing.  I can edit and preview the video in the app, but there is never any output.

**Comment by WyattBlue** [comment]
I wish I read your second comment sooner since I spent a good bit time looking at the xml output on window to see if there any problems. Now I know that the xml export is fine but I have no clue what the real root issue is. Oh well, maybe rewriting some component will fix this in the future at some point.
(You video looks like it has nothing out of the ordinary)

Also, thoughtfully responding to all your issues takes time away from actually fixing app bugs. In the future, I would like for you to just privately email me.

**Comment by BroBryce64** [comment]
So, just for fun, I attempted to use the .14 version instead of .17 of the app.  

This time, it shut down when I went to the open file dialog.  A couple of seconds later, I got a notification from Windows Defender saying it was sending the file for analysis.  This got me thinkin, so I checked in my Avast antivirus log, and it turns out it was raising red flags there, too.  But I have my Avast set on silent mode since it's got lots of annoying pop-ups all the time.  It didn't quarantine it, but apparently it was ringing bells.  

So I disabled Windows Defender and set an exception for both the .14 and .17 folders, and located the MP4 file both inside and outside those folders, and it output the XML and media files perfectly.

So now, I guess the next question is, what is triggering antivirus with this app?  Is there a solution to that?
